problems/sevenBoom.js

```diff
@@ -11,8 +11,19 @@
 * 17 is also replaced with 'BOOM' because it contains a 7.
 */
 
-function sevenBoom() {
-
+function sevenBoom(n) {
+    let arr = []
+    let  i = 1
+    while(i <= n){
+        if(((i % 7) === 0) || (i % 10) === 7){
```

Comment:
Consider what would happen to the numbers 70-79. Would you receive the word "BOOM" with every number that contains 7?